examples/deprecated-jenkins-inline-scan-v1/Jenkinsfile

```diff
@@ -0,0 +1,39 @@
+pipeline{
+    agent {
+        docker {
+            image 'docker:stable'
+        }
+    }
+    environment {
+        IMAGE_NAME = 'docker.io/alpine:3.7'
+        SYSDIG_SECURE_URL = 'https://secure.sysdig.com'
+        SYSDIG_SECURE_TOKEN = 'Dummy-Secure-Token'
+    }
+    stages {
+        stage('Build / Pull Image') {
+            steps {
+                script {
+                    deployRegistry = "https://registry.name/sysdig"
+                    pullSecret = 'dummy-pull-secret'
+                    deployRegistryCredentialId = 'ID'
+
+                    docker.withRegistry(deployRegistry, deployRegistryCredentialId) {
+                        script {
+                            sh "docker pull ${IMAGE_NAME}"
+                        }
+                        
+                    }
+                }
+            }
+        }
+        stage('Scan') {
+            steps {
+                sh "docker run --rm -v /var/run/docker.sock:/var/run/docker.sock sysdig/sysdig-inline-scan -s ${SYSDIG_SECURE_URL} -k ${SYSDIG_SECURE_TOKEN} ${IMAGE_NAME}"
```

Comment:
- Use `secure-inline-scan` instead of `sysdig-inline-scan`?
- Pin to `secure-inline-scan:1`?

Reply:
:1 does not exist... and sysdig/sysdig-inline-scan does not exist, apparently. So we should use sysdiglabs/secure-inline-scan:latest instead. Let me fix this and the ci-demo/Jenkinsfile too. Thanks!

Reply:
Should be fixed now.